### DISCUSSION TITLE
File content as environment variable

### DISCUSSION THREAD

**Discussion by laur1s** [cloudreach/sceptre#717] [Q&A]
Hello,
I have a following sceptre configuration: 

    template_path: ecs-service.j2
    ENV: !environment_variable test
    AppDnsName: !stack_output_external SCEPTRE-ECS-{{ stack_group_config.Environment }}-route53::{{ environment_variable.listener }}
    SSMStoreAppPath: {{ stack_group_config.Environment }}/RDS/TEST
    sceptre_user_data:
       env:
      {% include 'app.yml' %}
      {% include {{ environment_variable.test }}.yml %}

My use case is that I want to launch multiple ecs services and use configurations eg app1, app2 and use yaml files for storing app specific  configurations. Is it possible to do this natively in sceptre?

**Comment by laur1s** [comment]
I found out that there is a --var-file parameter `--var-file <var_file>
A YAML file of variables to template into config files.` But I can't find no documentation om how to actually use it

**Comment by nvengal** [comment]
There is also the `file_contents` resolver. https://sceptre.cloudreach.com/latest/docs/resolvers.html
